src/main/webapp/resources/css/makeRentPost.css

```diff
@@ -0,0 +1,41 @@
+#map {
+  height: 100%;
+}
+#navbarul {
+  list-style-type: none;
+  margin: 0 auto;
+  padding-left: 0px;
```

Comment:
Do we need to specify values with 0 for padding-left, top? I think 0 is implicit, so can be omitted

Reply:
Removing the two values seems to shift the nav bar to the right and down, so I will keep for now.